cmd/cut-release/main.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/plaid/go-envvar/envvar"
+)
+
+type envVars struct {
+	// Version is the new release version to use
+	Version string `envvar:"VERSION" default:"development"`
+}
+
+func main() {
+	env := envVars{}
+	if err := envvar.Parse(&env); err != nil {
+		log.Fatal(err)
+	}
+
+	updateHardCodedVersions(env.Version)
+
+	generateTypescriptClientDocs()
+}
+
+func generateTypescriptClientDocs() {
+	// Run `yarn install` to make sure `TypeDoc` dep is installed
+	cmd := exec.Command("yarn", "install")
```

Comment:
I think we should use the `--frozen-lockfile` here. We aren't adding any new dependencies and we don't want to make any changes to the lockfile in a commit that should solely be responsible for changing the version numbers and updating the documentation.